src/game/components/RotationAngle.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef GAME_COMPONENTS_ROTATIONANGLE_HPP_
+#define GAME_COMPONENTS_ROTATIONANGLE_HPP_
+
+#include "ecs/Component.hpp"
+
+namespace game
+{
+    struct RotationAngle : public ecs::Component {
+        float rotationAngle;
+
+        RotationAngle(float protationAngle) : rotationAngle(protationAngle) {}
```

Comment:
`protationAngle`  that sounds dirty...
Try `pRotationAngle`, also use camel case on component parameters, this would be easier to read.